schemas/2014-04-01/Microsoft.Sql.json

```diff
@@ -1539,6 +1697,52 @@
       ],
       "description": "Microsoft.Sql/servers/administrators"
     },
+    "servers_auditingPolicies_childResource": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "oneOf": [
+            {
+              "type": "string"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
```

Comment:
Understood, but it's generated by AutoRest. Does it make a material difference to customer experience? If yes, we should file a bug on autorest. If no, let's leave it.